elastic/kibana
# [Security Solution] [Bugfix] Fixes broken alert actions (add to case, investigate in timeline)

## Summary

In `EventDetailsFooter` on the call for `useFetchEcsAlertsData` was failing. However, we already can pass this data from the `onClick` that opens the event details from row data (onEventDetailsPanelOpened) and then we don't need the call. EDIT: we left the call as backup as it is needed for case details request

Additionally, the `normalizedEventFields` needed some backup as we are passing an empty array for the `data` property on add alert to case props? 

The missing Ecs data surfaced the following bugs:

Resolves https://github.com/elastic/kibana/issues/108900
Resolves https://github.com/elastic/kibana/issues/108901

To test:

1. On the alert page, add an alert to case from the `...` context menu. Go to the case and confirm the alert added looks like the second alert and not the first from the below image:
  <img width="979" alt="Screen Shot 2021-08-19 at 2 48 06 PM" src="https://user-images.githubusercontent.com/6935300/130141848-5741ff68-aca5-427d-8735-2ca687988342.png">

2. Back on the alert page, expand an alert
  a. confirm add to case works ([unlike here](https://github.com/elastic/kibana/issues/108900))
  b. confirm inspect in timeline works ([unlike here](https://github.com/elastic/kibana/issues/108901))


### Checklist

Delete any items that are not applicable to this PR.
